### DISCUSSION TITLE
Dexcom G6 pairing lost after being away for a long time

### DISCUSSION THREAD

**Discussion by dagord** [NightscoutFoundation/xDrip#2468] [Q&A]
Hello,
I'm reporting this, which I don't know if it's a real issue or an expected behaviour.

I was away from my phone (Xiaomi Redmi note 10 running MIUI 13.0.3) for more than 60 minutes; when I returned xDrip said "signal loss", and on the system status page it said something like "hunting transmitter"; my Dexcom G6 transmitter disappeared also from the paired bluetooth devices list.

I tried restarting collector, reboot phone and others, but the only solution was changing the trasmitter id and reverting back: the phone asked for pairing and all restarted working again.

I'm using xDrip+ 2122080314 but I suspect it's an issue with the phone itself, since with my old one this never happened.

**Comment by jamorham** [comment]
please update to the latest nightly and see if the problem persists
